### PR TITLE
Add css modules configuration

### PR DESCRIPTION
**Pull request by Nasdan**

### CODE REVIEW

**Comment by Nasdan** [comment]
This is a first review, using css-modules without any intelliSense.

I found a webpack loader [https://github.com/olegstepura/typed-css-modules-loader](https://github.com/olegstepura/typed-css-modules-loader) to use like preLoader in webpack configuration.
```javascript
module: {
    preLoaders: [
      {
        test: /\.css$/,
        exclude:/node_modules/,
        loader: 'typed-css-modules'
      }
    ],
...
```

It creates a d.ts file for each css when we run app. So, it seems like you have to run app twice to create d.ts files or create a command to build css first and then run app.

Do you think that it makes sense for us to include typed-css-modules?

**Comment by brauliodiez** [comment]
Can you open another branch so we can compare both approaches? Include @linuxonrails in this review

Are test passing? No error?

**Comment by Nasdan** [comment]
Ok, I'm going to create another branch.

I don't found any spec for this component, so I'm going to create it.

**Comment by brauliodiez** [comment]
Can you open another branch so we can compare both approaches? Include @linuxonrails in this review

Are test passing? No error?

**Comment by brauliodiez** [comment]
One more thing it would be a good idea to work with scss and include a global theme scss just implement a proof of concept with one var value consumes by the local scss

File: karma.conf.js [hunk 1]
```diff
@@ -27,6 +28,12 @@ module.exports = function (config) {
             {
                 test: /\.json$/,
                 loader: 'json'
+            },
+            {
+              test: /\.css$/,
+              exclude:/node_modules/,
+              //NOTE: Avoid import like [name]__[local]___[hash:base64:5] to create a well known class name
+              loader: ExtractTextPlugin.extract('style','css?modules&importLoaders=1&localIdentName=[local]')
```
Comment (Nasdan) [inline karma.conf.js]: This is the key to create a well known css class name where we are testing.

If we use `[name]__[local]___[hash:base64:5]` like in dev environment we are going to expect unique values like `styles__pageError404__XXX` where `XXX` is the hash value and this value could be changed

**Comment by brauliodiez** [comment]
I think we should go ahead with CSS Modules + SASS, we should create a simple wiki page to let other mates know how ti add their modules. On the other had we should make a session to explain them to the team :).

**Comment by Nasdan** [comment]
Move to PR #67

**Pull request closed without merging**